SOP/SOP-push.adoc

```diff
@@ -59,61 +464,318 @@ NOTE: It will use the values mapped in the Secret created by the operator with t
+deploymentconfig.apps.openshift.io/example-unifiedpushserver-postgresql   1          1         1         image(postgresql:10)
 
-== General procedure
+NAME                                                         DOCKER REPO                                               TAGS      UPDATED
+imagestream.image.openshift.io/ups-imagestream               172.30.1.1:5000/unifiedpush/ups-imagestream               latest    12 days ago
+imagestream.image.openshift.io/ups-oauth-proxy-imagestream   172.30.1.1:5000/unifiedpush/ups-oauth-proxy-imagestream   latest    12 days ago
 
-. Capture a snapshot of the 'UnifiedPush Server' Grafana dashboard and track it over time. The metrics can be useful for identifying performance issues over time.
+NAME                                                                   HOST/PORT                                                                      PATH      SERVICES                                      PORT      TERMINATION   WILDCARD
+route.route.openshift.io/example-unifiedpushserver-unifiedpush-proxy   example-unifiedpushserver-unifiedpush-proxy-unifiedpush.192.168.64.27.nip.io             example-unifiedpushserver-unifiedpush-proxy   <all>     edge/None     None
+----
 
-. Capture application logs for analysis.
-.. Get the pod names by running `oc get pods`
-.. Save the logs by running `oc logs <database-podname> > <filename>.log` for each pod
+=== Optional configurations
 
-NOTE: You are able to the the logs by the Console (OCP UI) as well.
+==== Monitor
 
-. If necessary, recreate the service pod to restore service.
-. Check the operator pod is present as it is responsible for managing the service pod as described in https://github.com/aerogear/unifiedpush-operator/blob/0.1.0/SOP/SOP-operator.adoc[UnifiedPushOperatorDown]
+If the https://github.com/aerogear/unifiedpush-operator#monitoring-service-metrics[Monitoring Service (Metrics)] is enabled for your installation you are able to check its Grafana Dashboard, `UnifiedPush Server`, and the Prometheus Monitoring instance used for it.
+
+==== Backup
 
+. Switch to the UPS namespace by running `oc project <namespace>`. E.g `oc project unifiedpush`
+. Check that link:./deploy/crds/push_v1alpha1_unifiedpushserver_cr_with_backup[UnifiedPushServerWithBackup CR] is deployed in the same namespace as the operator by running `oc get UnifiedPushServer`. Following the expected result.
++
+NOTE: Just one kind of UnifiedPushServer CR can be applied, however, if the backup service is enable for your installation then it means that it is using the ink:./deploy/crds/push_v1alpha1_unifiedpushserver_cr_with_backup[UnifiedPushServerWithBackup CR].
++
+[source,shell]
+----
+$ oc get UnifiedPushServer
+NAME                        AGE
+example-unifiedpushserver   9d
+----
++
+. To ensure that it is the UnifiedPushServer with the Backup see its specs by running `oc describe UnifiedPushServer`.
```

Comment:
Hi @grdryn the info is not the same. When has the bkp the data in the CR is not the same when it has not. Why `oc get ups -o yaml` instead of `oc describe UnifiedPushServer`? What is the advantage? IMHO: `oc describe UnifiedPushServer` make more clear what is required to do here and more intuitive as well.

Reply:
Ok fine, but what's the advantage of running `oc get UnifiedPushServer` in the code block on L670 then?

The intention of my original comment on this line was more like "there's room for improvement here, one way or the other".

Reply:
Note that the idea here is to describe for who never saw the project and has no idea over it. So, IHMO makes more sense.

1. Get the object
2. If has the object
3. Then check if is with the bkp or not ( the data changed and some example over what is expected can be helpful in this scenario to troubleshooting ). 

I hope that we can agree to move forward here in this way. If not, could write here as you wish then I will cut and past.